index.html

```diff
@@ -1,3 +1,44 @@
-﻿
+<!DOCTYPE html>
+<html>
+<body>
+  <img src="img/cookies.jpg" alt="Cookies">
+<h1>The Best Chocolate Chip Cookies</h1>
 
+<h3>INGREDIENTS</h3>
+<p>
+  <ulstyle ="list-style-type:circle;">
```

Comment:
On your page (https://lisakirsh.github.io/2019-spring-01h-recipe-page/) you can see that the tabbing for this section is flush against the left side of the page instead of being indented like the instructions. That's because you have `ulstyle` all as one word. The browser is expecting `<ul style='list-style-type: circle;'>` where the `ul` is the tag name and `style` is an attribute of that tag.